tests/SbrpTests/PathUtilities.cs

```diff
@@ -0,0 +1,26 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.IO;
+using System.Runtime.CompilerServices;
+
+namespace SbrpTests;
+
+internal static class PathUtilities
+{
+    public static string GetRepoRoot () =>
+        (string) AppContext.GetData ("SbrpTests.RepoRoot")!;
+
+    public static string GetSourceBuildRepoRoot ()
+    {
+        var artifactsDir = (string) AppContext.GetData ("SbrpTests.ArtifactsDir")!;
+        return Path.Combine (artifactsDir, "source-build", "self", "src");
+    }
+    
+    public static string GetSourceBuildPackagesShippingDir ()
+    {
+        var configuration = (string) AppContext.GetData ("SbrpTests.Configuration")!;
+        return Path.Combine (GetSourceBuildRepoRoot(), "artifacts", "packages", configuration, "Shipping");
+    }
```

Comment:
nit: remove the extra spaces
```suggestion
    public static string GetRepoRoot () =>
        (string)AppContext.GetData("SbrpTests.RepoRoot")!;

    public static string GetSourceBuildRepoRoot ()
    {
        var artifactsDir = (string)AppContext.GetData("SbrpTests.ArtifactsDir")!;
        return Path.Combine(artifactsDir, "source-build", "self", "src");
    }
    
    public static string GetSourceBuildPackagesShippingDir ()
    {
        var configuration = (string)AppContext.GetData("SbrpTests.Configuration")!;
        return Path.Combine(GetSourceBuildRepoRoot(), "artifacts", "packages", configuration, "Shipping");
    }
```

Reply:
I was following the style of https://github.com/dotnet/runtime/blob/a9791e69991ad542d6bbf1d9a8e77127210499bb/src/tools/illink/test/Trimming.Tests.Shared/PathUtilities.cs#L29C1-L30. Should I match how runtime formats this or should I remove the spaces?

Reply:
I've honestly never seen that style used before. It's not used throughout the runtime repo. My suggestion is to remove the spaces.

Reply:
This runtime code originated from dotnet/linker. That style is weird, don't use it.